Classifiers/classifier-Prisma_Cloud.json

```diff
@@ -14,9 +14,521 @@
                 "b1acdeff-4959-4c14-8a5e-2adc1016a3d5": "AWS IAM Policy Misconfiguration",
                 "ef7c537b-72eb-42a7-bab7-cb2d22c76a0d": "AWS IAM Policy Misconfiguration",
                 "f53107a2-00b2-46fb-98a9-1f12262c7d44": "AWS IAM Policy Misconfiguration",
-                "fd4dae57-509e-4374-96d3-e136821fc3f3": "AWS IAM Policy Misconfiguration"
+                "fd4dae57-509e-4374-96d3-e136821fc3f3": "AWS IAM Policy Misconfiguration",
+                "14d10ad2-51df-4b07-be69-e94951cc7067": "AWS EC2 Instance Misconfiguration",
+                "2378dbf4-b104-4bda-9b05-7417affbba3f": "AWS EC2 Instance Misconfiguration",
+                "2dbda57f-33d4-459a-97ae-dec7e81f9ec4": "AWS EC2 Instance Misconfiguration",
+                "3b642d25-4534-487a-9399-c2622754ecb5": "AWS EC2 Instance Misconfiguration",
+                "519456f2-f9eb-407b-b32d-064f1ac7f0ca": "AWS EC2 Instance Misconfiguration",
+                "520308c5-57e3-4061-b9bf-1ce5325a2d61": "AWS EC2 Instance Misconfiguration",
+                "5599b97c-2965-4fd2-9370-927c368abd2d": "AWS EC2 Instance Misconfiguration",
+                "566686e8-0581-4df5-ae22-5a901ed37b58": "AWS EC2 Instance Misconfiguration",
+                "65daa6a0-e040-434e-aca3-9d5765c96e7c": "AWS EC2 Instance Misconfiguration",
+                "6eaf6455-1659-4c4b-bff5-c8c7b0fda201": "AWS EC2 Instance Misconfiguration",
+                "760f2823-997e-495f-a538-5fb073c0ee78": "AWS EC2 Instance Misconfiguration",
+                "89cbc2f1-fcb0-48b9-be71-4cbe2d18a5f7": "AWS EC2 Instance Misconfiguration",
+                "8dd9e369-0c09-4477-97a2-ff0d50507fe2": "AWS EC2 Instance Misconfiguration",
+                "a9f1b983-f216-486e-b8ea-7259764fc420": "AWS EC2 Instance Misconfiguration",
+                "ab7f8eda-18ab-457c-b5d3-fd4f53c722bc": "AWS EC2 Instance Misconfiguration",
+                "ab8b6bb8-a730-4bdf-a4d5-080c01e97335": "AWS EC2 Instance Misconfiguration",
+                "b82f90ce-ed8b-4b49-970c-2268b0a6c2e5": "AWS EC2 Instance Misconfiguration",
+                "c2074d5a-aa28-4dde-90c1-82f528cec55e": "AWS EC2 Instance Misconfiguration",
+                "cdcd663c-e9c9-4472-9779-e5f38751524a": "AWS EC2 Instance Misconfiguration",
+                "ee03a420-89d6-4745-a0ac-98878cb56cf4": "AWS EC2 Instance Misconfiguration"
 	},
 	"mapping": {
+                "AWS EC2 Instance Misconfiguration": {
+                        "dontMapEventToLabels": false,
+                        "internalMapping": {
+                                "Account": {
+                                        "complex": {
```

Comment:
You shouldn't use the account field anywhere since it's not supposed to exist anymore, please update in other places as well